### PR TITLE
Added --list-versions CLI option to list the endpoint versions

### PR DESCRIPTION
**Pull request by afshinhm-ericsson**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -135,8 +135,10 @@ dependencies {
     providedRuntime("org.springframework.boot:spring-boot-starter-tomcat")
 
     //Injectable Message Library and its Implementation
-    compile 'com.github.Ericsson:eiffel-remrem-shared:0.1.4'
-    compile ('com.github.Ericsson:eiffel-remrem-semantics:0.1.7')
+    //compile 'com.github.Ericsson:eiffel-remrem-shared:0.1.5'
+    compile files ('/home/evasiba/git/eiffel-remrem-shared/build/libs/remrem-shared-0.1.5.jar')
```
Comment (p-backman-ericsson) [inline build.gradle]: You need to remove references to local files.

File: src/main/java/com/ericsson/eiffel/remrem/generate/cli/CLIOptions.java [hunk 2]
```diff
@@ -62,6 +63,7 @@ public static Options createCLIOptions() {
         contentGroup.addOption(new Option("json", "json_content", true, "json content"));              
         options.addOptionGroup(contentGroup);
 
+        options.addOption("lv", "list-versions", false, "list the version and all loaded protocols");
```
Comment (p-backman-ericsson) [inline src/main/java/com/ericsson/eiffel/remrem/generate/cli/CLIOptions.java]: list_versions We use underscore in all other cases.

File: src/main/java/com/ericsson/eiffel/remrem/generate/controller/EiffelMessagingVersionsController.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.ericsson.eiffel.remrem.shared.VersionService;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonParser;
+
+@RestController @RequestMapping("/eiffelmessagingversions")
```
Comment (p-backman-ericsson) [inline src/main/java/com/ericsson/eiffel/remrem/generate/controller/EiffelMessagingVersionsController.java]: I think it is enough to call the endpoint "/versions".

File: build.gradle (hunk 1 above)
Comment (afshinhm-ericsson) [inline build.gradle]: I'll remove them but we will not merge this until we have released new version of eiffel-remrem-shared, and eiffel-remrem-semantic

**Pull request merged**